MSAL/src/native_auth/public/state_machine/delegate/SignInDelegates.swift

```diff
@@ -44,28 +44,30 @@ public protocol SignInPasswordStartDelegate {
 
     /// Notifies the delegate that the sign in operation completed successfully.
     /// - Parameter result: An object representing the signed in user account.
-    @MainActor func onSignInCompleted(result: MSALNativeAuthUserAccountResult)
+    /// - Note: If a flow requires a code but this optional method is not implemented, then ``onSignInPasswordError(error:)`` will be called.
```

Comment:
I think this docc comment is wrong because it shouldn't mention requiring a code, should it? I think it is better to use the generic "Note: If a flow requires this optional method and it is not implemented..." text that you used in other cases.

Reply:
Good catch, updated in every place it was happening

Reply:
This is now solved